### PR TITLE
[MINI-5485, MINI-5695] Missing events for open / close permission settings

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Add pause and resume event for permissions and share preview for MiniAppSingleView

**Video**
https://rak.box.com/s/esfpcslakehe4imd7d1j7wnflt312q4v

## Links
https://jira.rakuten-it.com/jira/browse/MINI-5485

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Review by Climbatize — APPROVED** [review]
LGTM, but let's be careful when making comparisons of equatables

**Review by GondiTilak — APPROVED** [review]
Looks good to me :+1:

File: Example/Controllers/SwiftUI/Features/Single/MiniAppSingleView.swift [hunk 1]
```diff
@@ -179,9 +193,13 @@ struct MiniAppSingleView_Previews: PreviewProvider {
...
 
-struct MiniAppPermissionRequest: Identifiable {
+struct MiniAppPermissionRequest: Identifiable, Equatable {
     let id = UUID()
     let sdkConfig: MiniAppSdkConfig
     let info: MiniAppInfo
     let manifest: MiniAppManifest
+
+    static func == (lhs: MiniAppPermissionRequest, rhs: MiniAppPermissionRequest) -> Bool {
+        return lhs.id == rhs.id
```
[the remark this answers is not in the document]
Comment (laubengaier) [reply Example/Controllers/SwiftUI/Features/Single/MiniAppSingleView.swift:203]: @Climbatize imho I think this adds just some boilerplate comparison. `MiniAppPermissionRequest` is only used to fulfill `onChange` need for `MiniAppSingleView` as for handling the events so there is no actually need for a real comparison if it's used as the current intention. Can you explain where you see a need for a full compare of the requests like `permRequest1 == permRequest2` in a different scenario hence it's single use case?

**Pull request merged**